common/lib/xmodule/xmodule/tests/test_capa_module.py

```diff
@@ -361,6 +361,65 @@ def test_showanswer_past_due(self):
                                             graceperiod=self.two_day_delta_str)
         self.assertFalse(still_in_grace.answer_available())
 
+    def test_showanswer_after_attempts(self):
+        """
+        With showanswer="after_attempts",
+        the show answer button should
+        only appear after the user has
+        attempted the problem(i.e clicked the
+        submit button) for the requisite number of times.
+
+        Once the button appears, it does not disappear again.
+        """
+        # Cannot see answer because user has not attempted
+        # answering question for the required number of
+        # times.
+        before_attempts_with_max = CapaFactory.create(showanswer="after_attempts",
```

Comment:
Multiple lines like this shouldn't be aligned under the parenthesis;
please follow the formatting specified here:
https://github.com/stvstnfrd/edx-platform/blob/7c709518474446d00c782b1e5e37c3137f45d9d6/openedx/stanford/style/python/collections-multiline.markdown#explanation